.github/workflows/tests.yml

```diff
@@ -145,7 +145,7 @@ jobs:
     #if: ${{ env.NIGHTLY }}
     # in the meantime, copy-paste this:
     if: ${{ github.event_name == 'schedule' || github.event_name == 'push' || github.event_name == 'workflow_dispatch' }}
-    runs-on: ubuntu-18.04  # use an older ubuntu's kernel to closer emulate the older Debian
+    runs-on: ubuntu-20.04
```

Comment:
I'm going to also point a manual run of `tests.yml` at this PR's branch so that I can preemptively test the remaining OSs, too (i.e. the ones that aren't tested by default on PRs).

Reply:
Everything here passed, although we're getting a warning here:

```
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: actions/checkout, actions/checkout
```

I presume this is just a matter of going from `actions/checkout@v2` to `actions/checkout@v3`